core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -95,59 +88,57 @@
       optional(8, "dateCol", DateType.get()),
       required(9, "timeCol", TimeType.get()),
       required(10, "timestampCol", TimestampType.withoutZone()),
-      optional(11, "uuidCol", UUIDType.get()),
-      required(12, "fixedCol", FixedType.ofLength(4)),
-      required(13, "binaryCol", BinaryType.get())
+      required(11, "fixedCol", FixedType.ofLength(4)),
```

Comment:
Sorry if I'm missing some context, but why is the `uuidCol` dropped?

Reply:
UUID columns aren't currently supported in the Parquet writers for Iceberg generics. Rather than adding that support here, I thought it was better to remove it. We can add it back later, but I don't want to block ORC metrics on it.

Reply:
SGTM. Thanks for the context.